packages/core/src/components/screens/AboutScreen.tsx

```diff
@@ -1,6 +1,63 @@
 import React from "react";
-import { Typography } from "@equinor/mad-components";
+import { Cell, EDSStyleSheet, Typography, useStyles } from "@equinor/mad-components";
+import { ScrollView, View } from "react-native";
+import { useMadConfig } from "../../store/mad-config";
 
 export const AboutScreen = () => {
-    return <Typography>Not implemented yet</Typography>;
+    const config = useMadConfig();
+    const styles = useStyles(themeStyles)
+    const environment = config.environment.charAt(0).toUpperCase() + config.environment.slice(1);
+    
+    return (
+        <ScrollView
+            contentInsetAdjustmentBehavior="automatic"
+            contentContainerStyle={styles.contentContainer}>
+                <Cell>
+                    <View style={styles.cellContainer}>
+                        <Typography variant="h2">Client</Typography>
+                        <View style={styles.rowContainer}>
+                            <View style={styles.columnContainer}>
+                                <Typography>Configuration</Typography>
+                                <Typography>BuildNr</Typography>
+                                <Typography>App version</Typography>
+                            </View>
+                            <View style={styles.columnContainer}>
+                                <Typography>{environment}</Typography>
+                                <Typography>{config.about?.buildNumber}</Typography>
+                                <Typography>{config.appVersion}</Typography>
+                            </View>
+                        </View>
+                    </View>
+                </Cell>
+                <Cell>
+                    <View style={styles.cellContainer}>
+                        <Typography variant="h2">Api</Typography>
+                        <View style={styles.columnContainer}>
+                            <Typography>Endpoints</Typography>
+                            <View>
+                                {config.about?.endpoints.map(endpoint => (
+                                    <Typography key={endpoint}>{endpoint}</Typography>
+                                ))}
```

Comment:
One more thing. The app doesn't necessarily know about the mad api. I think it should still be displayed here, however. You can get it from `getMadCommonBaseUrl()`. I suggest removing it in `Chronicles` config too, to avoid duplicate entries

Reply:
Good point 👌 I added `madCommonBaseUrl` as a default endpoint in [5dfa832](https://github.com/equinor/mad/pull/310/commits/5dfa83200627b50cd6967b76caaae94784c5476d).